stan/math/prim/functor/broadcast_array.hpp

```diff
@@ -23,14 +24,11 @@ class broadcast_array {
 
   /** \ingroup type_trait
    * We can assign any right hand side which allows for indexing to a
-   * broadcast_array. The idea is that the entry for the first index is what
-   * gets assigned. The most common use-case should be where the rhs is some
-   * container of length 1.
+   * broadcast_array. The idea is that the argument is summed.
```

Comment:
[optional]
The original doc was very poorly written here.  Doc should not be in first person (plural or singular).  It should not say "The idea is that X"---it should just say X.  A compound like "use-case" should only be hyphenated when used as an adjective.

Reply:
Fixed.